### PR TITLE
cdc: add docs about the claim check.

### PR DESCRIPTION
**Pull request by ran-huang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/15074
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by 3AceShowHand — APPROVED** [review]

File: ticdc/ticdc-sink-to-kafka.md [hunk 1]
```diff
@@ -298,16 +321,16 @@ Starting from v7.3.0, TiCDC Kafka sink supports sending only the handle keys whe
...
-# This configuration is introduced in v7.3.0.
-# Empty by default, which means when the message size exceeds the limit, the changefeed fails.
-# If this configuration is set to "handle-key-only", when the message size exceeds the limit, only the handle key is sent in the data field. If the message size still exceeds the limit, the changefeed fails.
-large-message-handle-option = "handle-key-only"
+# large-message-handle-option is introduced in v7.3.0.
+# Defaults to "none". When the message size exceeds the limit, the changefeed fails.
+# When set to "handle-key-only", if the message size exceeds the limit, only the handle key is sent in the data field. If the message size still exceeds the limit, the changefeed fails.
+# When set to "claim-check", if the message size exceeds the limit, the message is sent to external storage.
+large-message-handle-option = "claim-check"
+claim-check-storage-uri = "s3://claim-check-bucket"
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:333]:
```suggestion
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 2]
```diff
@@ -298,16 +321,16 @@ Starting from v7.3.0, TiCDC Kafka sink supports sending only the handle keys whe
...
 ```toml
 [sink.kafka-config.large-message-handle]
-# This configuration is introduced in v7.3.0.
-# Empty by default, which means when the message size exceeds the limit, the changefeed fails.
-# If this configuration is set to "handle-key-only", when the message size exceeds the limit, only the handle key is sent in the data field. If the message size still exceeds the limit, the changefeed fails.
-large-message-handle-option = "handle-key-only"
+# large-message-handle-option is introduced in v7.3.0.
+# Defaults to "none". When the message size exceeds the limit, the changefeed fails.
+# When set to "handle-key-only", if the message size exceeds the limit, only the handle key is sent in the data field. If the message size still exceeds the limit, the changefeed fails.
+# When set to "claim-check", if the message size exceeds the limit, the message is sent to external storage.
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:331]:
```suggestion
```

Comment (Oreoxmt) [reply ticdc/ticdc-sink-to-kafka.md:331]: https://github.com/pingcap/docs-cn/blob/9cc5a36bc3d4ae81797a66372a894cb47ea8b833/ticdc/ticdc-sink-to-kafka.md?plain=1#L327-L335

```toml
[sink.kafka-config.large-message-handle]
# 该参数从 v7.3.0 开始引入
# 默认为空，即消息超过大小限制后，同步任务失败
# 设置为 "handle-key-only" 时，如果消息超过大小，data 字段内容只发送 handle key；如果依旧超过大小，同步任务失败
large-message-handle-option = "handle-key-only"
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 3]
```diff
@@ -351,3 +374,74 @@ When a Kafka consumer receives a message, it first checks the `onlyHandleKey` fi
...
+If the message contains the `claimCheckLocation` field, the Kafka consumer reads the large message data stored in JSON format according to the address provided by the field. The message format is as follows:
+
+```json
+{
+  key: "xxx",
+  value: "xxx",
+}
+```
+
+key and value contain the encoded large message, which should have been sent to the corresponding field in the Kafka message. Consumers can parse the data in these two parts to restore the content of the large message.
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:447]:
```suggestion
The `key` and `value` fields contain the encoded large message, which should have been sent to the corresponding field in the Kafka message. Consumers can parse the data in these two parts to restore the content of the large message.
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**